website/server/controllers/api-v3/challenges.js

```diff
@@ -410,13 +547,18 @@ api.exportChallengeCsv = {
 };
 
 /**
- * @api {put} /api/v3/challenges/:challengeId Update a challenge
+ * @api {put} /api/v3/challenges/:challengeId Update a challenge. Only reflects
+ * changes to the full name of the challenge. Other challenge details cannot be
+ * updated through this route.
```

Comment:
With this route, I was only able to update the 'name' field of the challenge. Updates to 'shortName', 'group', and 'prize' failed silently. Let me know if this should be accepting updates to other fields.